test/gtest/uct/uct_test.cc

```diff
@@ -1241,23 +1249,15 @@ void uct_test::entity::connect_to_ep(unsigned index, entity& other,
     ucs_status_t status;
     uct_ep_h ep, remote_ep;
     uct_ep_params_t ep_params;
-    uct_iface_addr_t *iface_addr;
 
     reserve_ep(index);
     if (m_eps[index]) {
         return; /* Already connected */
     }
 
     other.reserve_ep(other_index);
-
-    iface_addr = (uct_iface_addr_t*)ucs_alloca(other.iface_attr().iface_addr_len);
-
-    ep_params.field_mask = UCT_EP_PARAM_FIELD_IFACE |
-                           UCT_EP_PARAM_FIELD_IFACE_ADDR;
+    ep_params.field_mask = UCT_EP_PARAM_FIELD_IFACE;
```

Comment:
can we do both - create ep with iface addr and fo connect_to_ep_v2

Reply:
maybe in this case we can check whether it is different in the connect and return an error?

Reply:
ok, done